test/test.options.ts

```diff
@@ -44,108 +51,162 @@ describe('Options', () => {
     assert.equal(google._options.hello, 'world');
   });
 
-  it('should promote endpoint options over global options', () => {
+  it('should promote endpoint options over global options', (done) => {
     const google = new googleapis.GoogleApis();
     google.options({hello: 'world'});
     const drive = google.drive({version: 'v2', hello: 'changed'});
-    const req = drive.files.get({fileId: '123'}, Utils.noop);
-    assert.equal(req.hello, 'changed');
+    createNock('/drive/v2/files/123');
+    drive.files.get({fileId: '123'}, (err, res) => {
+      if (err) {
+        return done(err);
+      }
+      assert.equal(res.config.hello, 'changed');
+      done();
+    });
   });
 
   it('should support global request params', (done) => {
     const google = new googleapis.GoogleApis();
     google.options({params: {myParam: '123'}});
     const drive = google.drive('v2');
-    let req = drive.files.get({fileId: '123'}, Utils.noop);
-    // If the default param handling is broken, query might be undefined, thus
-    // concealing the assertion message with some generic "cannot call .indexOf
-    // of undefined"
-    let query = req.uri.query || '';
-    assert.notEqual(
-        query.indexOf('myParam=123'), -1, 'Default param not found in query');
-    nock.enableNetConnect();
-    Utils.loadApi(google, 'drive', 'v2', {}, (err, d) => {
-      nock.disableNetConnect();
-      if (err) {
-        return done(err);
-      }
-      req = d.files.get({fileId: '123'}, Utils.noop);
+    createNock('/drive/v2/files/123?myParam=123');
+    drive.files.get({fileId: '123'}, (err, res) => {
+      if (err) return done(err);
       // If the default param handling is broken, query might be undefined, thus
       // concealing the assertion message with some generic "cannot call
       // .indexOf of undefined"
-      query = req.uri.query || '';
+      let query = Utils.getQs(res) || '';
       assert.notEqual(
           query.indexOf('myParam=123'), -1, 'Default param not found in query');
-      done();
+      nock.enableNetConnect();
```

Comment:
@ofrobots for some reason, I can't seem to get enableNetConnect() to work here.  I keep getting a `Error: Nock: No match for request`, which AFAICT shouldn't happen.  I could use a second look from someone here.